src/pages/research/projects/dayu.mdx

```diff
@@ -0,0 +1,297 @@
+
+
+<p style={{ position: 'relative', width: '200px', height: '150px', overflow: 'hidden' }}>
+  <img
+    src={require("@site/static/img/projects/dayu/dayu-logo2.png").default}
+    style={{ position: 'absolute', width: '280px', height: '150px' }}
+  />
+</p>
+
+# DaYu: Optimizing Workflow Performance by Elucidating Semantic Data Flow
+
+<ProjectBadges projectId="dayu" />
+
+Distributed scientific workflows nowadays face challenges in data movement through storage systems. 
+DaYu employs careful runtime measurement, maps domain semantics to low-level I/O operations, and utilizes 
+effective visualization and analysis of semantic data flows to understand how semantic datasets move through storage.
+
+## Introduction
+
+High-Performance Computing (HPC) workflows are evolving with increasing data intensity. 
+These workflows are growing in complexity, featuring multiple stages encompassing simulation, analysis, 
+and AI applications with diverse data demands. 
+Data transfer between HPC tasks currently relies on shared storage layers like Parallel File System (PFS) 
+and Burst Buffer, which can suffer from slow access and I/O contention.
+
+Enhancing data movement within workflows poses a significant challenge. 
+Strategic task scheduling, data caching, and staging have emerged as 
+effective means to boost I/O performance by reducing computation wait times. 
+With more I/O expertise, one can also utilize and fine-tune various I/O
+libraries, middleware, and file system configurations. The tuning often 
+requires iterative testing and are limited to specific workload, whille other 
+workloads still experiencing high latency with shared storage.
+
+Understanding I/O behavior is imperative when deciding on the correct strategies to enhance data access. 
+Details about data access within workflow tasks can effectively guide improvements in I/O and system configuration. 
+Existing application I/O profiling tools lack the ability to provide analyze of data access behavior across 
+tasks in a workflow and capture semantic
+information related to its low-level I/O requests. Such tools would
+be valuable for providing more straightforward insights into
+data access patterns across multiple tasks. By filling this gap, we
+can develop better methods for managing data movement and
+optimizing the overall workflow.
+
+**In this work, we unveil a fresh workflow optimization perspective with**
+- careful runtime measurement of data access metrics,
+- recovering the mapping of domain semantics to low-level I/O operations, and
+- effective visualization and analysis of semantic data flows for the complete workflow.
+
+## Background
+
+<center>
+  <p style={{ position: 'relative', width: '800px', height: '220px', overflow: 'hidden' }}>
+    <img
+      src={require("@site/static/img/projects/dayu/graphs/hdf5_structure.png").default}
+      style={{ position: 'absolute', left: '-40px', top: '-200px', right: '-140px', width: '800px', height: '600px' }}
```

Comment:
Absolute positioning is dangerous as it might not get rendered correctly is different devices (mobile/desktop). Please include your image using the Markdown notation.

Reply:
This goes for the other images as well, please fix all of the images on the page.

Reply:
all image uses the same format as other project pages (I tried but the caption with markdown format is not showing).